extensions/3DTILES_bounding_volume_S2/README.md

```diff
@@ -0,0 +1,459 @@
+</p>
+
+```json
+"boundingVolume": {
+  "extensions": {
+    "3DTILES_bounding_volume_S2": {
+      "token": "89c6c7",
+      "minimumHeight": 0,
+      "maximumHeight": 1000
+    }
+  }
+}
+```
+
+The following example illustrates usage of `3DTILES_bounding_volume_S2`:
+
+```json
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "geometricError": 1000000,
+  "extensionsUsed": [
+    "3DTILES_bounding_volume_S2"
+  ],
+  "extensionsRequired": [
+    "3DTILES_bounding_volume_S2"
+  ],
+  "root": {
+    "boundingVolume": {
+      "extensions": {
+        "3DTILES_bounding_volume_S2": {
+          "token": "3",
+          "minimumHeight": 0,
+          "maximumHeight": 1000000
+        }
+      }
+    },
+    "refine": "REPLACE",
+    "geometricError": 50000,
+    "children": [
+      {
+        "boundingVolume": {
+          "extensions": {
+            "3DTILES_bounding_volume_S2": {
+              "token": "2c",
+              "minimumHeight": 0,
+              "maximumHeight": 500000
+            }
+          }
+        },
+        "refine": "REPLACE",
+        "geometricError": 500000,
+        "children": [
+          {
+            "boundingVolume": {
+              "extensions": {
+                "3DTILES_bounding_volume_S2": {
+                  "token": "2f",
+                  "minimumHeight": 0,
+                  "maximumHeight": 250000
+                }
+              }
+            },
+            "refine": "REPLACE",
+            "geometricError": 250000,
+            "children": [
+              {
+                "boundingVolume": {
+                  "extensions": {
+                    "3DTILES_bounding_volume_S2": {
+                      "token": "2ec",
+                      "minimumHeight": 0,
+                      "maximumHeight": 125000
+                    }
+                  }
+                },
+                "refine": "REPLACE",
+                "geometricError": 125000
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+## Implicit Subdivision
+
+When used with [`3DTILES_implicit_tiling`](../3DTILES_implicit_tiling), a `QUADTREE` subdivision scheme will follow the rules for subdivision as defined by the S2 cell hierarchy. When an `OCTREE` subdivision scheme is used, the split in the vertical dimension occurs at the midpoint of the `minimumHeight` and `maximumHeight` of the parent tile.
+
+| Cell  | Quadtree Subdivision | Octree Subdivision |
+|---|---|---|
+| ![Parent Cell](figures/parent.png)  | ![Quadtree Cells](figures/quadtree.png)  | ![Octree Cells](figures/octree.png)  |
+
+To ensure continuity of the Hilbert curve, the faces of the cube are rotated as shown in the diagram below. This must be carefully considered when interpreting the tile coordinates in implicit tiling, since the traversal order in the odd-numbered faces is the mirror of the order in even numbered faces.
+
+```
+                  ^---->  <----^
```

Comment:
@javagl I seem to recall you had a nicer diagram for this? could you export that so I can include it here?

Reply:
Yeah, I had created that because I thought that it might go into the overview, but that would be a bit too detailed for that section.


![3d-tiles-s2-faces](https://user-images.githubusercontent.com/5597569/139741021-35f64b5e-8670-42a9-afb4-5825c94bd32a.png)

The SVG source, for reference: [3d-tiles-s2-faces.zip](https://github.com/CesiumGS/3d-tiles/files/7456090/3d-tiles-s2-faces.zip)

(An aside: I *think* that GitHub can display SVGs directly, with some quirks. But for "portability", PNG is probably preferable...)